chapters/7-publication.tex

```diff
@@ -508,75 +475,74 @@ \subsection{Organizing code for reproducibility}
-DIME has adopted several practices and requirements to support the production
-of high-quality reproducibility packages by its researchers.
+DIME requires all academic outputs to successfully pass a computational reproducibility check 
+before being submitted for publication. 
+We have adopted several practices and requirements to support the production
+of high-quality reproducibility packages.
 The materials for these practices are publicly available,
 so you can use them to check the reproducibility of your own work.
-DIME code must be submitted for a computational reproducibility check
-before publication as a working paper or submission to a journal.
-This reproducibility check is initiated by contacting the DIME Analytics team
-and submitting the Reproducibility Package Checklist.
-In addition to this formal requirement for reproducibility review,
-DIME projects are required to be maintained continuously at high quality.
-These practices must include ensuring that project code is organized
-with a master script so that computational reproducibility is a one-click exercise.
+This reproducibility check is initiated by submitting the Reproducibility Package Checklist.\sidenote{
+	\url{https://github.com/worldbank/dime-standards/blob/master/dime-research-standards/checklists/reproducibility-package-checklist.pdf}}
+DIME projects are required to organize code
+with a master script, to facilitate handovers across team members 
+and make the computational reproducibility check a one-click exercise.
+Compliance with these and other coding standards at DIME is monitored through
+quarterly peer code review rounds, which allows research assistants to improve their code and documentation as it is written,
+rather than revisiting it in a rush near publication time.
 DIME projects are also expected to use Git and GitHub
 to document project work and collaboration,
-and to keep the master branch up-to-date as a working edition.
-Research assistants also participate in quarterly peer code review rounds
-so that they can improve their code and documentation as it is written,
-and not be required to revisit it in a rush near publication time.
+and to keep the master branch up-to-date as a working edition.\sidenote{
+	\url{https://github.com/worldbank/dime-standards/tree/master/dime-research-standards/pillar-3-research-reproducibility}}
 
 % ensuring code reproducibility
-Before releasing the code, make sure it functions identically on a fresh install of your chosen software.
-A new user should have no problem getting the code to execute perfectly.
-In either a scripts folder or in the root directory,
-you should include a master script that allows the reviewer to run the entire project
-and re-create all raw outputsby changing only a single line of code:
-the one setting the directory path.
+Before publicly releasing a reproducibility package,
+it is essential to make sure that the code executes perfectly on a fresh install of your chosen software.
```

Comment:
```suggestion
it is essential to make sure that the code runs identically 
on your individual setup compared to 
a fresh installation of your software.

```
I do not think `perfectly` is the right word. This is the most explicit way to say what we mean